kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/security/CertificateListTabUi.ui.xml

```diff
@@ -50,17 +58,19 @@
                     ui:field="certificatesGrid" />
             </b:Row>
             <b:Modal title="Add Certificate" fade="true" dataBackdrop="STATIC" dataKeyboard="true" b:id="certAddModal" ui:field="certAddModal" closable="false">
-  				<b:ModalBody b:id="certAddModalBody">
-  				    <g:ScrollPanel ui:field="certAddModalBody" addStyleNames="{style.modal-scroll-panel}"/>
-  				</b:ModalBody>
-  				<b:ModalFooter>
-  					<b:Button dataDismiss="MODAL" b:id="closeModalButton" ui:field="closeModalButton">Close</b:Button>
-  					<b:Button type="PRIMARY" b:id="nextStepButton" ui:field="nextStepButton">Do something</b:Button>
-  				</b:ModalFooter>
-  			</b:Modal>
+                <b:ModalBody b:id="certAddModalBody">
+                    <g:ScrollPanel ui:field="certAddModalBody" addStyleNames="{style.modal-scroll-panel}"/>
+                </b:ModalBody>
+                <b:ModalFooter>
+                    <b:Button dataDismiss="MODAL" b:id="resetModalButton" ui:field="resetModalButton">Reset</b:Button>
+                    <b:Button type="PRIMARY" b:id="nextStepButton" ui:field="nextStepButton">Do something</b:Button>
```

Comment:
I kind of find more natural the old ordering of buttons: in the first view the next button should be on the right like in the following view where the confirmation with the submit button is in the right.
I find odd that in the first view we have, in order, next and close and in the following view reset, close, apply.

Reply:
6975c4d0efab4662ff7787a6afd856b100cdc0d2 revert the order as was before.